src/main/scala/cromwell/engine/workflow/WorkflowActor.scala

```diff
@@ -518,6 +524,23 @@ case class WorkflowActor(workflow: WorkflowDescriptor, backend: Backend)
   }
 
   /**
+    *  Publish workflow structure to centralized logging.
+    */
+  private def publishWorkflowEvent() : Unit = {
+    import org.json4s._
+    import org.json4s.native.Serialization.{write}
+
+    logger.info(s"--------> publishWorkflowEvent ")
+    implicit val formats = DefaultFormats + FieldSerializer[WorkflowExecution]()
+    implicit val actorSystem = context.system
+
+    for {
+      status <- globalDataAccess.getWorkflowExecution(workflow.id)
+    } yield PubSubMediator.publish(WorkflowExecutionEvent, write(status))
```

Comment:
WorkflowLogger is an entity for listening events only and will be unsubscribed either through api or if any chance actor got killed then eventstream do it automatically, however it listen from PubSubMeditor that will decouple components reporting log from component logging logs.